crates/assistant_tool/src/tool_schema.rs

```diff
@@ -36,6 +36,7 @@ fn adapt_to_json_schema_subset(json: &mut Value) -> Result<()> {
         }
 
         obj.remove("format");
+        obj.remove("additionalProperties");
```

Comment:
I'm getting similar errors for the [fetch](https://github.com/modelcontextprotocol/servers/tree/main/src/fetch) official MCP server, but for two other fields.

```suggestion
        obj.remove("additionalProperties");
        obj.remove("exclusiveMaximum");
        obj.remove("exclusiveMinimum");
```